addons/sale_loyalty/models/sale_order.py

```diff
@@ -361,12 +361,12 @@ def _get_reward_values_discount(self, reward, coupon, **kwargs):
             }]
         discount_factor = min(1, (max_discount / discountable)) if discountable else 1
         mapped_taxes = {tax: self.fiscal_position_id.map_tax(tax) for tax in discountable_per_tax}
+        name_translated = _('Discount: %(desc)s%(tax_str)s')
```

Comment:
this looks good to me

as a ugly alternative that would not remove the fallback system of `_`  we could do:

```suggestion
        def name_translated(self, desc, tax_str):
            return _('Discount: %(desc)s%(tax_str)s', desc, tax_str)
```
(to have the self in the underscore function call locals) but not sure if it's worth the hassle